src/barsPlus-directive.js

```diff
@@ -18,7 +18,7 @@
 import d3 from 'd3';
 import qvangular from 'qvangular';
 import ldwBarsPlus from './ldw-barsPlus';
-import './barsPlus.css';
+import './barsPlus.less';
 
```

Comment:
in another PR i added the parent's class name from the CSS class to the template which resulted the same fix.
thoughts about that?